src/middlewares/validators/createUserPayload.js

```diff
@@ -4,10 +4,10 @@ const createUserPayloadValidator = (req, res, next) => {
   const schema = Joi.object({
     name: Joi.string().alphanum().min(2).max(45).trim(true).required(),
     surname: Joi.string().alphanum().min(2).max(45).trim(true).required(),
-    middleName: Joi.string().alphanum().min(2).max(45).trim(true).required(),
+    middleName: Joi.string().alphanum().min(2).max(45).trim(true),
     password: Joi.string().min(8).max(100).required(),
-    phoneNumber: Joi.string().trim(true).length(10).pattern(/^\d+$/).required(),
-    address: Joi.string().trim(true).required(),
```

Comment:
Why these fields are not required? When we create user as usual, these fields must be required. It it is google user, let's use some custom values like "Phone number is not provided" etc.